src/admin/content/helpers/reducer.ts

```diff
@@ -1,22 +1,25 @@
 import { createReducer } from '../../../shared/helpers';
+import { ContentBlockConfig } from '../../content-block/content-block.types';
 
-import {
-	ContentEditBlocksAction,
-	ContentEditBlocksActionType,
-	ContentEditBlocksState,
-} from '../content.types';
+import { ContentEditAction, ContentEditActionType, ContentEditState } from '../content.types';
 
-export const CONTENT_EDIT_BLOCKS_INITIAL_STATE = (): ContentEditBlocksState => ({
-	cbConfigs: [],
+export const CONTENT_EDIT_INITIAL_STATE = (
+	cbConfigs: ContentBlockConfig[] = []
```

Comment:
waarom wrap je dit nog eens in een object?, kan je de array niet instellen als de state?

Reply:
Dit is de initial state die ik meegeef aan de useReducer op de content-edit view.
Ik volgde hierin gwn de principes van redux, waar je een state object hebt met daarin uw properties